src/components/BugzillaComponents/index.jsx

```diff
@@ -16,7 +16,7 @@ const styles = ({
   },
 });
 
-const sortByComponentName = (a, b) => a.label - b.label;
+const sortByComponentName = (a, b) => (a.label <= b.label ? -1 : 1);
```

Comment:
You can use [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) to compare strings.

Reply:
That's great to know! Thanks